### PR TITLE
RPY-1216: Create survey component with emoji, stars, and simple survey variants

### PR DESCRIPTION
**Pull request by lekhasholehati**
### Description
Create a new survey component base this requirement: [Link ZeroHeight ](https://zeroheight.com/19c7d85e4/p/480bdf-changelog/b/38c3bc).
The survey components have 3 variants that can be use for client, they are emoji survey, stars rating survey, and simple survey with 2 option.

### Changes

-  Add new folder survey in library (setup and initialize package.json)
-  Add import '@tunaiku-ui-web/survey/survey.scss' in core index.scss for load the module
- Add the documentation in survey.mdx.

### User Interface
screencapture-localhost-8000-components-survey-2023-02-16-15_21_55

### CODE REVIEW

File: library/survey/scss/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$amb-padding-content-bordered: 0px 2.5vw 0px 8px;;
+$amb-border-active: 2px solid #FDC440;
+$amb-content-gap: (
```
Comment (cakahlul) [inline library/survey/scss/_variables.scss:3]: ini kalau ukurannya ikutin yg ada di file ini gimana? https://github.com/tunaiku/tunaiku-ui-web/blob/RPY-1216/library/spacing/scss/_variables.scss .

atau kalau memungkinkan, kita re-use aja spacing variablenya.

- [ ] adjust gap value

File: library/survey/scss/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    margin-top: 0px;
+
+    @include amb-survey-content-bordered-active();
+  }
+}
+
+@mixin amb-survey-content-gap-size() {
+  @each $gap, $gap-value in $amb-content-gap {
+    &--#{$gap} {
+      gap: $gap-value !important;
```
Comment (cakahlul) [inline library/survey/scss/_mixins.scss:86]: kalau pakai margin aja gabisa kah?

- [ ] question

File: library/survey/scss/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  }
+}
+
+@mixin amb-survey-subtitle() {
+  &-subtitle {
+    @include amb-typography-style('caption12');
+    color: amb-color-theme('dark');
+
+    &--center {
+      font-weight: bold;
```
Comment (cakahlul) [inline library/survey/scss/_mixins.scss:33]: ini boldnya ga perlu ya. karena dari HTML nya bisa pakai class typography yang udah ada aja. kalau dipaksa bold gini nanti kurang dinamis jadinya. gimana?

- [ ] remove font-weight property (?)

**Review by cakahlul — CHANGES_REQUESTED** [review]
ada feedback. minta tolong cek dulu ya

File: library/survey/scss/_mixins.scss (hunk 2 above)
Comment (lekhasholehati) [reply library/survey/scss/_mixins.scss:86]: @cakahlul bisa mas, saya perbaiki, jadinya saya ga nambahin modifier --gapSmall langsung pakai margin aja di survey-content--bordered

File: library/survey/scss/_mixins.scss (hunk 3 above)
Comment (lekhasholehati) [reply library/survey/scss/_mixins.scss:33]: @cakahlul  oke mas bisa harusnya pakai class typography

**Review by cakahlul — APPROVED** [review]

**Review by Christian-007 — APPROVED** [review]

**Pull request merged**